bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -189,7 +189,7 @@ public void channelUnlinked(ChannelUID channelUID) {
     protected void validateConfigurationParameters(Map<String, Object> configurationParameters)
             throws ConfigValidationException {
         ThingType thingType = TypeResolver.resolve(getThing().getThingTypeUID());
-        if (thingType != null) {
+        if (thingType != null && thingType.getConfigDescriptionURI() != null) {
```

Comment:
The function would make sense if a null ConfigDescriptionURI is a valid one and differs from none ConfigDescriptionURI. I don't think this will be the case ever. ;)
I don't know which third party bundle are using the has... function.